docs/UserGuide.md

```diff
@@ -323,24 +359,26 @@ A person can have any number of tags and insurance policies (including 0).
 **Examples**:
```

Comment:
For this line: 
"To include the URL, remember to use '>' to indicate that..."
it might be good to use markdown code to show the > character, just like how we described in the "What information can we store for each client contact?" table.